### ISSUE TITLE
Troubleshooting Mode admin bar menu not keyboard operable

### ISSUE THREAD

**Issue by afercia** [WordPress/health-check#266]
The Troubleshooting Mode in the admin bar is not operable with a keyboard.

<img width="463" alt="screenshot 2019-02-23 at 22 21 18" src="https://user-images.githubusercontent.com/1682452/53291827-ced42d80-37b9-11e9-92a6-a9bc1a5f22b7.png">

Some of its menu items, including the top level one, aren't links so they can't be focused when tabbing with the keyboard.

All the items need to be actual links, so I guess some reorganization of the menu tree is necessary.

**Comment by Clorith** [comment]
Curious if creating links as `href="#"` are acceptable here, or would that in turn be confusing if you clicked a link and went nowhere?

If it's confusing I'm happy to link to their respective areas within wp-admin, which sort of makes sense, plugins and themes goes to those pages, the top level item takes you to the dashboard which has the widget explaining troubleshooting mode and such.

**Comment by afercia** [comment]
In core, we've been removing `href="#"` links over the past years because they're not links 🙂 They're used for developers convenience but they're still communicated to assistive technologies as links, while they actually behave like buttons.

https://core.trac.wordpress.org/query?keywords=~semantic-buttons

**Comment by Clorith** [comment]
Ahh, didn't know we were fixing that, but that is essentially the concern I had with doing that, so linking to the respectively correct pages is definitely what should be done here, thank you!

**Comment by Clorith** [comment]
Fixed in #279
